### PR TITLE
ReadmeFile

### PR DESCRIPTION
**Pull request by amitsinghsutara**
Hey @burrage ,
Here Is the Updated Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,442 @@
...
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+
+
+
+
+
```
Comment (burrage) [inline README.md:20]: @amitsinghsutara Can we link the English, French, Swahili, and Arabic versions here please?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,442 @@
...
+b.	Library development directory (for programmers only)
+
+20.	If everything is working correctly you should see the Interactive Content type when you click Add Content from the top menu.
+
+
+
+
+
+1. Got to the (C:\xampp\htdocs\drupal\sites\default\files\h5p\development),
```
Comment (burrage) [inline README.md:222]: Were these supposed to fall under another header that got removed...? "Cloning the project" perhaps?

Comment (burrage) [reply README.md:222]: Or perhaps they just got renumbered and should be 21, 22, etc.?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,442 @@
...
+
+
+  };
+
+export default gameData;
+```
+
+### Compile and build the new version
+
+Step1 : For Compiling newer versions go to the library.json file and increase the "patchVersion" by 1 in both the
```
Comment (burrage) [inline README.md:293]: Consistency with numbering and styling here-- continue 1. ... , 2. ... , etc. format used above.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,442 @@
...
+};
+
+
+export default ScoreBoard;
+```
+
+
+
+
+<!-- ROADMAP -->
```
Comment (burrage) [inline README.md:370]: Remove this whole "Roadmap" section.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,442 @@
...
+<!-- CONTACT -->
+## Contact
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- ACKNOWLEDGMENTS -->
+## Acknowledgments
+
```
Comment (burrage) [inline README.md:404]: Feel free to add the Sutara team here!

File: README.md [hunk 6]
```diff
@@ -1,2 +1,442 @@
...
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- CONTRIBUTING -->
+## Contributing
+
+
```
Comment (burrage) [inline README.md:382]: Can just leave a "Contribution guidelines coming soon" sentence here.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,442 @@
...
+<!-- LICENSE -->
+## License
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- CONTACT -->
+## Contact
+
```
Comment (burrage) [inline README.md:397]: Can add:

"If you'd like to contact us, please feel free to e-mail us at info@curiouslearning.org."

File: README.md [hunk 8]
```diff
@@ -1,2 +1,442 @@
...
+
+
+<p align="right">(<a href="#top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
+## License
+
```
Comment (burrage) [inline README.md:390]: Remove this "License" section as we have a license file already so this would be redundant information to keep updated.

**Review by burrage — COMMENTED** [review]
Reviewing README.md only. Not going to approve as I see there were other commits to code files tacked onto this commit for @Rajesh1041 to review...?

File: README.md (hunk 1 above)
Comment (amitsinghsutara) [reply README.md:20]: @burrage ,
 Sure will add,
 Thanks a lot for feedback

**Review by Rajesh1041 — CHANGES_REQUESTED** [review]
Keep only the readme file in this PR.

**Pull request closed without merging**